### PR TITLE
Set LocusID on shortcut and notification for on device intelligence

### PR DESCRIPTION
**Pull request by alabiaga**
Set LocusId on notification and associated shortcut for on device intelligence.
Also improves experience on People Widgets (https://developer.android.com/guide/topics/ui/conversations#conversation-widgets)

### CODE REVIEW

**Comment by alabiaga** [comment]
cc\ @alan-signal

File: app/src/main/java/org/thoughtcrime/securesms/notifications/v2/NotificationBuilder.kt [hunk 1]
```diff
@@ -87,6 +89,12 @@ sealed class NotificationBuilder(protected val context: Context) {
     }
   }
 
+  fun setLocusId(locusId: String) {
```
Comment (cody-signal) [inline app/src/main/java/org/thoughtcrime/securesms/notifications/v2/NotificationBuilder.kt:92]: No need to make this a special case, just roll it into the setShortcutId method

**Review by cody-signal — COMMENTED** [review]
Please don't tag devs on PRs. For example, Alan doesn't work on Android anymore.

**Comment by alex-signal** [comment]
@cody-signal I think Aaron meant to tag me, he's been in contact with me via email.

**Comment by cody-signal** [comment]
Sorry about that, I'll let you two figure it out.

**Comment by cody-signal** [comment]
Thanks, pulled in and will be out soon.

**Pull request closed without merging**